src/mockSdk.js

```diff
@@ -32,6 +32,18 @@ export const mockSDKPerson = {
   orgId: 'orgID',
 };
 
+export const mockSDKPersonList = {
```

Comment:
[`mockSDKPerson`](https://github.com/webex/sdk-component-adapter/blob/master/src/mockSdk.js#L25) already exists, no need to add another mock object